modules/test-runner/ops/webpack.config.js

```diff
@@ -1,20 +1,37 @@
+<<<<<<< HEAD
```

Comment:
I will look into getting the build fixed btw... this is just an artifact of getting this PR out there for us to collaborate on.